rails/README.md

```diff
@@ -3,6 +3,8 @@
 - Use private instead of protected when defining controller methods.
 - Name date columns with `_on` suffixes.
 - Name datetime columns with `_at` suffixes.
+- Back boolean concepts like deleted?, published?, or active? with timestamps.
+  This can be valuable when you need to know **when** something took place. [Time for A Boolean](https://github.com/calebhearth/time_for_a_boolean) provides a nice interface for this.
```

Comment:
Thoughts on being explicit about what "backing with a timestamp" means?
```suggestion
- Back boolean concepts like deleted?, published?, or active? with timestamps columns in the database `deleted_at`, `published_at`, `active_at`.
  This can be valuable when you need to know **when** something took place. [Time for A Boolean](https://github.com/calebhearth/time_for_a_boolean) provides a nice interface for this.
```